maestro-ios-driver/src/main/kotlin/xcuitest/installer/LocalXCTestInstaller.kt

```diff
@@ -132,33 +138,34 @@ class LocalXCTestInstaller(
             .inputStream.source().buffer().readUtf8()
             .trim()
 
-        if (!processOutput.contains(UI_TEST_RUNNER_APP_BUNDLE_ID)) {
-            logger.info("Not able to find ui test runner app, going to install now")
-            val tempDir = File(tempDir).apply { mkdir() }
-            val xctestRunFile = File("$tempDir/maestro-driver-ios-config.xctestrun")
-
-            logger.info("[Start] Writing xctest run file")
-            writeFileToDestination(XCTEST_RUN_PATH, xctestRunFile)
-            logger.info("[Done] Writing xctest run file")
-
-            logger.info("[Start] Writing maestro-driver-iosUITests-Runner app")
-            extractZipToApp("maestro-driver-iosUITests-Runner", UI_TEST_RUNNER_PATH)
-            logger.info("[Done] Writing maestro-driver-iosUITests-Runner app")
-
-            logger.info("[Start] Writing maestro-driver-ios app")
-            extractZipToApp("maestro-driver-ios", UI_TEST_HOST_PATH)
-            logger.info("[Done] Writing maestro-driver-ios app")
-
-            logger.info("[Start] Running XcUITest with xcode build command")
-            xcTestProcess = XCRunnerCLIUtils.runXcTestWithoutBuild(
-                deviceId,
-                xctestRunFile.absolutePath,
-                port
-            )
-            logger.info("[Done] Running XcUITest with xcode build command")
-        } else {
-            logger.info("UI test runner already installed and running")
+        if (processOutput.contains(UI_TEST_RUNNER_APP_BUNDLE_ID)) {
```

Comment:
This change I don't understand.
Can't we keep the already running xcuitest runner as we did before?